README.md

```diff
@@ -110,7 +115,193 @@ addSubview(container)
 container.constrainEdgesToMargins()
 ```
 
-### 2. UIColor extensions for WCAG 2.0 contrast ratio calculations
+#### Constrain size
+
+There are three convenience methods for constraining the size of a view.
+
+```swift
+// constrain a button's size to 44 x 44 (3 different ways)
+let button = UIButton()
+addSubview(button)
+
+button.constrainSize(CGSize(width: 44, height: 44))
+button.constrainSize(width: 44, height: 44)
+button.constrainSize(44)
+```
+
+#### Constrain center
+
+There is an Auto Layout convenience method for centering views.
+
+```swift
+// center a container view to its superview
+let container = UIView()
+addSubview(container)
+
+container.constrainCenter()
+
+// center a button horizontally
+let button = UIButton()
+addSubview(button)
+
+button.constrainCenter(.x)
+
+// align a button and a label vertically by their centers
+let button = UIButton()
+let label = UILabel()
+addSubview(button)
+addSubview(label)
+
+button.constrainCenter(.y, to: label)
+```
+
+#### Constrain aspect ratio
+
+There is an Auto Layout convenience method for constraining aspect ratio:
+
+```swift
+// constrain to a 16:9 aspect ratio
+mediaPlayer.constrainAspectRatio(16.0 / 9)
+
+// constrain to a 1:1 aspect ratio
+profileImage.constrainAspectRatio(1)
+```
```

Comment:
Future improvements suggestion,

We can define enum for aspect ratio, also define one more method for constrain which accept that enum.

Enum may contain cases for popular aspect ratio like 4:3, 16:9 etc. and keep one case( with associated value) to accept custom value.

Reply:
Noted. Thanks, @dharmik-yml!